test/unit/middleware/auth/sign_in_with_email_middleware_test.dart

```diff
@@ -32,15 +31,15 @@ void main() {
 
       // setup middleware and invoke the middleware
       final mut = SignInWithEmailMiddleware(mockAuthService);
-      mut(testStore, SignInWithEmail(), testDispatcher);
+      mut(fakeStore, SignInWithEmail(), testDispatcher);
 
       // check that the service's stream emits the expected action and the store
       // dispatched the same action
       mockAuthService
           .emailSignInStream(testEmail, testPassword)
           .listen(expectAsync1((action) {
             expect(action, equals(testAction));
-            expect(testStore.dispatchedActions.contains(testAction), true);
+            expect(fakeStore.dispatchedActions, contains(testAction));
```

Comment:
Changed to using the `contains` matcher which I think makes for an easier to read expectation.